src/components/App/App.jsx

```diff
@@ -64,21 +92,24 @@ const handleProductQuantity = (productName, operation) => {
 
   // edit item quantity
   //TODO: throw error if item didn't exist in products
-  if (operation === Operations.Add) {
```

Comment:
I was trying to figure out why my operations were not working, and I tried to see if the strict equality was the reason. Turns out it was because the `Symbol` function (I mentioned the fix in 5 comments above). I'll change it back since I think strict equality is better code